test_data/generated/envs/hello/manifest.toml

```diff
@@ -44,6 +44,12 @@ hello.pkg-path = "hello"
 #   echo "it's gettin' flox in here"
 # '''
 
+# The `[services]` section of the manifest allows you to define services.
+# Services defined here use the packages provided by the `[install]` section 
+# and any variables you've defined in the `[vars]` section or `hook.on-activate` script.
+[services]
+# command = "postgres"
```

Comment:
does this need to be updated?

Reply:
Yeah, it should be consistent with the changes in this branch, but I'm not sure that using a full template is load bearing for the tests. Any objections to stripping the comments and empty sections?